setup.py

```diff
@@ -35,7 +35,7 @@
     'check-manifest>=0.25',
     'coverage>=4.0',
     'isort>=4.2.2',
-    'jsonresolver[jsonschema]>=0.2.0',
+    'jsonresolver[jsonschema]>=0.2.1',
```

Comment:
Are you trying to fix the `JSONSCHEMAS_HOST` warning here? Because I don't think that this is the problem; The only dependency difference between a running build and a failed build is the version of Flask (compare https://travis-ci.org/inveniosoftware/invenio-jsonschemas/jobs/238764205#L259 and https://travis-ci.org/inveniosoftware/invenio-jsonschemas/jobs/238764224#L259).